### PR TITLE
workflows: add containers job

### PR DESCRIPTION
**Pull request by lucab**
This adds a new workflow building the fedora-based container image.

### CODE REVIEW

File: .github/workflows/containers.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-fedora-image:
+    name: "Build container image (fedora)"
```
Comment (bgilbert) [inline .github/workflows/containers.yml:12]: `Fedora`

File: .github/workflows/containers.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
```
Comment (bgilbert) [inline .github/workflows/containers.yml:15]: `Check out`

**Review by bgilbert — APPROVED** [review]
LGTM.  Tiny copyediting nits.

**Comment by lucab** [comment]
Nits fixed, force-pushed.

**Review by jlebon — APPROVED** [review]

**Review by bgilbert — APPROVED** [review]

**Pull request merged**